packages/augur-core/source/contracts/reporting/Universe.sol

```diff
@@ -467,7 +467,7 @@ contract Universe is IUniverse, CashSender {
     }
 
     function getRepMarketCapInAttoCashInternal(uint256 _attoCashPerRep ) private view returns (uint256) {
-        return reputationToken.totalSupply().mul(_attoCashPerRep).div(10 ** 18);
+        return reputationToken.getTotalTheoreticalSupply().mul(_attoCashPerRep).div(10 ** 18);
```

Comment:
This is unrelated but came up in discord discussion so figured it'd be good to get in one batch of contract changes